src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const styles = sourceString
+    .split(';')
+    .filter(Boolean)
+    .map((style) => style.trim());
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation to improve readability. Align the methods accordingly.